hoot-core/src/main/cpp/hoot/core/util/Settings.cpp

```diff
@@ -120,9 +120,7 @@ class JsonLoader
         continue;
       if (!_s->hasKey(name))
       {
-        LOG_WARN("Unknown JSON setting: (" << name << ")");
-        //  Don't allow unknown settings to be loaded
-        continue;
+        throw HootException("Unknown JSON setting: (" + name + ")");
```

Comment:
Didn't I already see these changes in another ticket?  These are already in `develop`.  Don't know if this will cause a conflict or just wash out in the merge.